src/main/java/FindTaskCommand.java

```diff
@@ -0,0 +1,43 @@
+import java.util.List;
+
+/**
+ * A Command to find tasks from the task list based on a keyword.
+ */
+public class FindTaskCommand extends Command {
+    private String keyword;
+
+    public FindTaskCommand(String keyword) {
+        this.keyword = keyword;
+    }
+
+    /**
+     * Executes the command to find tasks from the task list based on a keyword.
+     *
+     * @param tasks The task list.
+     * @param ui The ui that handles user output.
```

Comment:
This param does not exist anymore and needs to be removed from the Javadoc comment.